modules/account-lib/src/index.ts

```diff
@@ -65,7 +65,7 @@ export { Algo };
 import * as Atom from '@bitgo/sdk-coin-atom';
 export { Atom };
 
-import * as Osmo from '@bitgo/sdk-coin-osmo';
+import * as Osmo from '@bitgo/abstract-cosmos';
```

Comment:
why this? we won't have coin specifi module?

Reply:
Here we are using TxnBuilderFactory which we have moved in abstract-cosmos.

Reply:
Lets have it from the osmo module itself, have the `@bitgo/abstract-cosmos` exported from osmo and exported here. Please export transaction builder from the osmos module also for backward compatibility since devs will be expecting to have the transaction builder available in the coin specific module itself

Reply:
Yes, let's have the transaction/transaction builders in the respective chains. They can extend from abstract cosmos transaction/transaction builders